### ISSUE TITLE
Testing React Apps

### ISSUE THREAD

**Issue by HurricaneJames** [reactjs/react-rails#131]
What is the preferred method of testing react-rails based apps? I tried combining react-rails and browserify-rails in an attempt to get CommonJS and JEST working, but I failed... a lot. Any suggestions?

Thanks,
JB

**Comment by tabolario** [comment]
I've been messing with [Teaspoon](https://github.com/modeset/teaspoon) attempting to get it to work with the React testing addons. Right now I'm stuck on PhantomJS crashing no matter what bind polyfill I use though, so I gave up for the time being. Will report back here once I get some more time to look at it.

**Comment by HurricaneJames** [comment]
I completely forgot that I created this issue. I ended up figuring out how to make react-rails and browserify-rails work together. I wrote a whole article about it on [reactjsnews.com](https://reactjsnews.com/setting-up-rails-for-react-and-jest/). This setup allows our team to use JEST for unit testing. It also works fine with our cucumber tests, but those are run through selenium/FF instead of PhantomJS.
